docs/_Docs_ReputationMining.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reputation Mining Client
+section: Docs
+order: 6
+---
+
+The reputation mining client is usable for testing but it has limited functionality. Still, it is possible to run a single miner instance for usable reputation scores on a local test network.
```

Comment:
This sounds like our miner is a a bit lame :) What's the "limited functionality" you are referring to here?